resources/domain/balance/index.ts

```diff
@@ -62,3 +62,8 @@ export const sortByTotalValue = (a: DisplayedBalance, b: DisplayedBalance) => {
 export function isNativeCurrency (address: string) {
   return address === NATIVE_CURRENCY
 }
+
+export function toTokenId (token: Token|Balance ) {
```

Comment:
might be a little more flexible to create a type or interface like
```
type WithTokenId {
  address: string
  chainId: number
}
```

and then have both `Balance` and `Token` types use that just to make sure it's all consistent. then this function can just take that type